src/ansys/fluent/core/session.py

```diff
@@ -32,6 +32,16 @@ def parse_server_info_file(filename: str):
     return ip, port, password
 
 
+def _get_preferences(session):
```

Comment:
@hpohekar it's possible that this is part of a repeating pattern that could be factored out